docs/_articles/en/v3.x/release-information.md

```diff
@@ -6,6 +6,39 @@ lang: en
 ### To update the plugin, please follow the linked [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
 
+## [v3.4.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.4.0) (08-17-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.3.0...v3.4.0)
+
+### Release Summary
+* Improvements to `eslint`:
+	- Execute `eslint` engine with babel parser to accommodate more modern javascript syntax
+	- Identify javascript target files that were analyzed by both `eslint` and `eslint-lwc`, and throw warning about the possibility of duplicate violations
+* Survey request banner fix
+* SFGE: Minor code fixes
+* Updates to local RetireJS Vulnerability Repository
+* PMD upgrade to 6.48.0
+
+**Closed issues:**
+
+- \[BUG\] UnexpectedException on v3 DFA on large ISV codebase [\#739](https://github.com/forcedotcom/sfdx-scanner/issues/739)
```

Comment:
I would preface each of these with FIX:

FIX: Large ISV codebases no longer throw `UnexpectedException` on v3.0 DFA.
FIX: `stdout` is in proper JSON format when `--format json` is used.
FIX: The `apex.jorje.parser.impl.BaseApexLexer` had been deduped.

Reply:
For future reference: Teresa and I had an offline conversation and decided that while we keep this intact to reflect the title in the original issue, we should modify the title while handling the issue to better reflect the actual problem we are fixing.